client/llb/source.go

```diff
@@ -452,6 +452,45 @@ func Differ(t DiffType, required bool) LocalOption {
 	})
 }
 
+func OCILayout(contentStoreIDAndHash string, opts ...OCILayoutOption) State {
+	gi := &OCILayoutInfo{}
+
+	for _, o := range opts {
+		o.SetOCILayoutOption(gi)
+	}
+	attrs := map[string]string{}
+	if gi.SessionID != "" {
+		attrs[pb.AttrLocalSessionID] = gi.SessionID
+		addCap(&gi.Constraints, pb.CapSourceLocalSessionID)
```

Comment:
Not LocalSessionID but define new cap for OCILayouts.

Reply:
Oh nice catch. That one slipped by me. Fixing.